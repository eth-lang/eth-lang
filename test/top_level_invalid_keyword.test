(* some loose int maybe? *)
66
