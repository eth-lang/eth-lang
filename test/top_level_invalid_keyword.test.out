test/top_level_invalid_keyword.test:2:0 Expected keyword let or import at top level, got "66" of type 'number'
